test/integrations/requirejs/test-requirejs.js

```diff
@@ -2,6 +2,18 @@ describe('Require.JS Pages', function() {
     it('loads mParticle properly', function() {
         var server = new MockHttpServer();
         server.start();
+        window.fetchMock.post('https://jssdks.mparticle.com/v3/JS/test_key/events', 200);
```

Comment:
Shouldn't we put this in with other urls?

Reply:
Good call

Reply:
seems like i'm having issues importing this from the src/utils file.  It's complaining about when I try to do a require or import statement.  I'll make a sus eng ticket for now.   https://mparticle-eng.atlassian.net/browse/SQDSDKS-5314